packages/atlas/src/api/hooks/nfts.ts

```diff
@@ -1,10 +1,82 @@
-import { useGetNftQuery } from '@/api/queries'
+import {
+  VideoCategoryWhereInput,
+  VideoOrderByInput,
+  useGetNftQuery,
+  useGetNftsQuery,
+  useGetVideosConnectionQuery,
+} from '@/api/queries'
+import { createLookup } from '@/utils/data'
 
-export const useNft = (videoId: string) => {
-  const { data, ...rest } = useGetNftQuery({ variables: { videoId: videoId }, skip: !videoId })
+export const useNft = (id: string) => {
+  const { data, ...rest } = useGetNftQuery({ variables: { id }, skip: !id })
 
   return {
-    nft: data?.ownedNfts[0],
+    nft: data?.ownedNftByUniqueInput,
+    ...rest,
+  }
+}
+
+type ChannelNftsOpts = {
+  category?: VideoCategoryWhereInput | null
+  sortNftsBy?: VideoOrderByInput
+}
+
+export const useChannelNfts = (channelId: string, opts?: ChannelNftsOpts) => {
+  const {
+    data: videosConnectionData,
+    loading: videosConnectionLoading,
+    ...rest
+  } = useGetVideosConnectionQuery({
+    variables: {
+      orderBy: opts?.sortNftsBy,
+      where: {
+        channel: { id_eq: channelId },
+        nft: {
+          metadata_contains: '',
+        },
```

Comment:
This is meant to say "NFT exists"?

Reply:
yeah, tested it out and it seems to be working. I'll add here a small comment as well.